lib/utils/messages.ts

```diff
@@ -45,5 +45,9 @@ Sample registry-secrets YAML file:
 		username: '_json_key'
 		password: '{escaped contents of the GCR keyfile.json file}'
 
-If an option is not specified, and a secrets.yml or secrets.json file exists in
-the balena directory (usually $HOME/.balena), this file will be used instead.`;
+For an example using the Google Container Registry, check this page:
+https://gist.github.com/pdcastro/d9739b9b81ad75ab5957bd5b8b405080
+
+If the --registry-secrets option is not specified, and a secrets.yml or
+secrets.json file exists in the balena directory (usually $HOME/.balena),
+this file will be used instead.`;
```

Comment:
Replaced "an option" with "the --registry-secrets option" because, when the help is rendered with e.g. "balena help push", the paragraph is printed on its own, and I've added a new paragraph before this one. (One would still reason that "secrets.yml" most likely applies to "--registry-secrets", but spelling it reduces ambiguity.)